Resources/Prototypes/_Sunrise/Entities/Clothing/Hands/gloves.yml

```
@@ -159,3 +158,28 @@
     fiberMaterial: fibers-insulative
     fiberColor: fibers-blue
   - type: FingerprintMask
+
+- type: entity
+  parent: [ ClothingHandsGlovesColorBlack, BaseMajorContraband ]
+  id: ClothingHandsGlovesCombatCQC
+  name: combat gloves
+  description: These high-grade tactical gloves are fireproof and shock resistant.
+  components:
+  - type: Sprite
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)